netaudio/dante/channel.py

```diff
@@ -86,6 +90,9 @@ def volume(self):
     def volume(self, volume):
         self._volume = volume
 
+    def matches_name(self, name_to_match):
+        return name_to_match == self.friendly_name or name_to_match == self.name
```

Comment:
```suggestion
        return name_to_match.lower() in [self.friendly_name.lower(), self.name.lower()]
```

[Audinate's documentation](https://dev.audinate.com/GA/dante-controller/userguide/webhelp/content/device_and_channel_names.htm) states that names should be compared as case insensitive.